docs/source/tutorial/create_a_pipeline.md

```diff
@@ -0,0 +1,329 @@
+
+If you want the model input table data to be saved to file (in `data/03_primary`) rather than used in memory, add an entry to `conf/base/catalog.yml`:
+
+```yaml
+model_input_table:
+  type: pandas.ParquetDataSet
+  filepath: data/03_primary/model_input_table.pq
+```
+
+### Test the example
+
+To test the progress of the example:
+
+```bash
+kedro run
+```
+
+You should see output similar to the following:
+
+<details>
+<summary><b>Click to expand</b></summary>
+
+```bash
+[08/09/22 17:00:54] INFO     Kedro project kedro-tutorial                                         session.py:346
+[08/09/22 17:01:10] INFO     Reached after_catalog_created hook                                     plugin.py:17
+                    INFO     Loading data from 'companies' (CSVDataSet)...                   data_catalog.py:343
+                    INFO     Running node: preprocess_companies_node:                                node.py:327
+                             preprocess_companies([companies]) -> [preprocessed_companies]
+                    INFO     Saving data to 'preprocessed_companies' (MemoryDataSet)...      data_catalog.py:382
+                    INFO     Completed 1 out of 3 tasks                                  sequential_runner.py:85
+                    INFO     Loading data from 'shuttles' (ExcelDataSet)...                  data_catalog.py:343
+[08/09/22 17:01:25] INFO     Running node: preprocess_shuttles_node: preprocess_shuttles([shuttles]) node.py:327
+                             -> [preprocessed_shuttles]
+
+                    INFO     Saving data to 'preprocessed_shuttles' (MemoryDataSet)...       data_catalog.py:382
+                    INFO     Completed 2 out of 3 tasks                                  sequential_runner.py:85
+                    INFO     Loading data from 'preprocessed_shuttles' (MemoryDataSet)...    data_catalog.py:343
+                    INFO     Loading data from 'preprocessed_companies' (MemoryDataSet)...   data_catalog.py:343
+                    INFO     Loading data from 'reviews' (CSVDataSet)...                     data_catalog.py:343
+                    INFO     Running node: create_model_input_table_node:                            node.py:327
+                             create_model_input_table([preprocessed_shuttles,preprocessed_companies,
+                             reviews]) -> [model_input_table]
+[08/09/22 17:01:28] INFO     Saving data to 'model_input_table' (MemoryDataSet)...           data_catalog.py:382
+[08/09/22 17:01:29] INFO     Completed 3 out of 3 tasks                                  sequential_runner.py:85
+                    INFO     Pipeline execution completed successfully.                             runner.py:89
+                    INFO     Loading data from 'model_input_table' (MemoryDataSet)...        data_catalog.py:343
+```
+</details>
+
+## Checkpoint
+
+This is an excellent place to take a breath and summarise what you have done so far.
```

Comment:
I really like this! 

Perhaps include something like this :) 
https://unsplash.com/photos/0fy6aJoPRhc

Reply:
Sold!